web/src/message_list.js

```diff
@@ -109,7 +109,7 @@ export class MessageList {
 
         if (interior_messages.length > 0) {
             this.view.rerender_preserving_scrolltop(true);
-            return true;
+            return {need_user_to_scroll: true};
```

Comment:
Please put this in a separate prep commit -- it seems like a meaningful bug fix?

Reply:
moved to [here](https://github.com/zulip/zulip/pull/28373#discussion_r1440941278)